src/panel/pages/events/components/TimelineSourceIcon.tsx

```diff
@@ -14,7 +14,7 @@ export const TimelineSourceIcon = styled.div<{
   transition: background-color 150ms ease-out;
 
   :before {
-    content: "${({ kind }) => kind[0].toUpperCase()}";
+    content: "${({ kind }) => kind && kind[0].toUpperCase()}";
```

Comment:
Was this causing an issue?

Looking at the [type definition](https://github.com/FormidableLabs/urql-devtools/blob/aa7047103f33d9d99edfd5f7ec410f9173673dff/src/panel/pages/events/components/TimelineSourceIcon.tsx#L4) this condition is always truthy

Reply:
Every time I open a RN app with the electron urql devtools I see this error on the Events page. I can open a different issue if you can reproduce it as well. 

![image](https://user-images.githubusercontent.com/12056467/115864824-1f01b580-a440-11eb-8480-30c706b66a47.png)

Reply:
Hmmm - I don't think this is a bug here, it sounds like we're ending up with invalid data.

Are you able to try and catch the event that we're sourcing the `kind` prop from. A console log [somewhere around here](https://github.com/FormidableLabs/urql-devtools/blob/649077c5daef98d8b0c1ecb6d769a686a7bfc36f/src/panel/pages/events/Timeline.tsx#L163) should help us work out why we're missing the needed property

Reply:
If you want, lets revert the fix here for now so we can get this PR merged and then make a separate PR/issue for this particular problem!

It's a good find either way 🚀  we need to be able to trust our types so lets find the source 🕵️

Reply:
Reverted, I will investigate more. Thanks @andyrichardson